### PR TITLE
Multiple UI fixes and improvements

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
PLT-4640 - Iphone 5s web view dropdown styling.
PLT-4547 - Updating few transitions.
PLT-4498 - Fixed flag icon overlapping on long names.
PLT-4619 - Scrollbar now visible in System console logs.
PLT-4694 - Autocomplete styling now visible on mobile.
PLT-4731 - Flagged posts link on mobile closes sidebars now.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4640
https://mattermost.atlassian.net/browse/PLT-4547
https://mattermost.atlassian.net/browse/PLT-4498
https://mattermost.atlassian.net/browse/PLT-4619
https://mattermost.atlassian.net/browse/PLT-4694
https://mattermost.atlassian.net/browse/PLT-4731

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [ ] All new/modified APIs include changes to the drivers
- [ ] Has enterprise changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/platform/blob/master/i18n/en.json) and [.../webapp/i18n/en.json](https://github.com/mattermost/platform/tree/master/webapp/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

### CODE REVIEW

File: webapp/components/sidebar_header_dropdown.jsx [hunk 1]
```diff
@@ -125,6 +126,11 @@ export default class SidebarHeaderDropdown extends React.Component {
         });
     }
 
+    switchTeam(teamUrl) {
+        this.setState({showDropdown: false});
```
Comment (jwilander) [inline webapp/components/sidebar_header_dropdown.jsx]: Need a `e.preventDefault()` in here somewhere or we're going to add `#` to the end of the URL

File: webapp/components/sidebar_header_dropdown.jsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ export default class SidebarHeaderDropdown extends React.Component {
         this.handleAboutModal = this.handleAboutModal.bind(this);
         this.aboutModalDismissed = this.aboutModalDismissed.bind(this);
         this.toggleAccountSettingsModal = this.toggleAccountSettingsModal.bind(this);
+        this.switchTeam = this.switchTeam.bind(this);
```
Comment (enahum) [inline webapp/components/sidebar_header_dropdown.jsx]: with the Team Sidebar this is obsolete

Comment (asaadmahmood) [inline webapp/components/sidebar_header_dropdown.jsx]: Well once the team sidebar is in, we might need to remove the switching links from the menu as well, so we can remove that from there as well.

Comment (enahum) [inline webapp/components/sidebar_header_dropdown.jsx]: they are removed in the Team Sidebar PR :)

but hey I can just rebase and remove this later on if this makes it to master before team sidebar

Comment (asaadmahmood) [inline webapp/components/sidebar_header_dropdown.jsx]: Oh, didn't know the PR was in. I'll remove this one then.

**Comment by asaadmahmood** [comment]
Hey @jwilander removed that code since it won't be needed with the teams sidebar.

**Pull request merged**